tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -119,14 +120,16 @@ stages:
             runOnce: 
               deploy:
                 steps: 
-                  - checkout: none
+                  - checkout: azure-sdk-build-tools
```

Comment:
You don't need to checkout. You just need to add a reference. Example https://github.com/Azure/azure-sdk-tools/blob/2e646542c4f21d64f493ef1e1b893a4489a2ecf0/eng/pipelines/templates/stages/1es-redirect.yml#L9